flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/catalog/hive/util/HiveTableUtil.java

```diff
@@ -71,4 +85,69 @@ public static TableSchema createTableSchema(List<FieldSchema> cols, List<FieldSc
 
 		return columns;
 	}
+
+	/**
+	 * Convert partition value string to actual type in flink.
+	 * @param partitionValue
+	 * @param clazz
+	 * @return
+	 */
+	//todo: check whether hive partition field can be decimal or not .
+	public static Object getActualObjectFromString(String partitionValue, Class clazz) {
```

Comment:
Maybe we can have a more meaningful method name, like getPartitionValueOjectFromString.